anndata/tests/test_base.py

```diff
@@ -95,6 +95,13 @@ def test_create_from_df_with_obs_and_var():
         AnnData(df, var=var.reset_index())
 
 
+def test_matching_int_index():
+    ad = AnnData(
```

Comment:
```suggestion
    adata = AnnData(
```

Since `ad` is often used as an abbreviation for the package, and it's nice when tests can be copy and pasted into a notebook/ repl.